### DISCUSSION TITLE
Why would a new install have alpha and beta history?

### DISCUSSION THREAD

**Discussion by mc12345678** [zencart/zencart#6388] [Q&A]
Day one of new release, download from the main page of https://zen-cart.com, via https://github.com/zencart/zencart/archive/refs/tags/v2.0.0.zip. Select to install a new store. Review database information and see that although this is a brand new install, there is history indicating that the main files are beta1 and the database as alpha1. 

Go into the master branch of the software and while the code reflects just being version 2.0.0, the database of a new install seemingly incorrectly identifies a history to alpha. 

This is bound to cause issues for users new to Zen Cart when upgrading to ~~2.0.1~~ 2.0.0a or similar when zc_install is not required/not expected to be used. History would show that it was "done" in the past (upgrade within a primary version), so obviously should be done again, right?

Correction applied about version upgrade as for some reason 2.0.0/2.0.1 seemed so different from 1.5.8/1.5.8a. *rolling eyes*

**Comment by scottcwilson** [comment]
It's a bug.  It only occurs for new stores.  It has been fixed.

**Comment by scottcwilson** [comment]
https://github.com/zencart/zencart/pull/6387

**Comment by mc12345678** [comment]
> It's a bug. It only occurs for new stores. It has been fixed.

> #6387

No, it still has not been fixed. 

This line of "correction" does not fix the issue as identified in the original discussion. Specifically:
https://github.com/zencart/zencart/blob/54c5acdc94e41518ae49993cf5767acfb489b321/zc_install/sql/install/mysql_zencart.sql#L3499

Which still has `alpha1` in the database information:
> INSERT INTO project_version_history (project_version_id, project_version_key, project_version_major, project_version_minor, project_version_patch, project_version_comment, project_version_date_applied) VALUES (2, 'Zen-Cart Database', '2', '0.0-alpha1', '', 'New Installation-v200', now());

**Comment by mc12345678** [comment]
Corrected by (not yet associated to a PR) commit: https://github.com/zencart/zencart/commit/47c45a32bf88b5153826368970061701cd31cdde

**Comment by drbyte** [comment]
Corrected by the re-tagged 2.0.0 release.
https://github.com/zencart/zencart/releases/tag/v2.0.0

**Comment by scottcwilson** [comment]
Thanks for catching this @mc12345678. I have added YET MORE reminders to the build docs to triple check these values.
